provider/mem/mem.go

```diff
@@ -100,37 +101,52 @@ func NewAlerts(ctx context.Context, m types.Marker, intervalGC time.Duration, al
 		logger:    log.With(l, "component", "provider"),
 		callback:  alertCallback,
 	}
-	a.alerts.SetGCCallback(func(alerts []*types.Alert) {
```

Comment:
`SetGCCallback` and `Run` method on Alerts store are now unused. Should those be deleted?

Reply:
I think `Run` is called by Inhibitor but `SetGCCallback` is unused in AM but since it's a public method, deleting it might break other things